docker-compose.yml

```diff
@@ -86,8 +89,10 @@ services:
       - "5005:8080"
     restart: always
     depends_on:
-      - oracle-data-api
-      - coms
+      oracle-data-api:
+        condition: service_started # no health check on oracle-data-api
```

Comment:
There actually is a health/status check, but I don't think this has ever been configured in OpenShift.

https://oracle-data-api-0198bb-dev.apps.silver.devops.gov.bc.ca/actuator/health

Reply:
I just pushed a change to enable heath check in docker compose `curl -f http://localhost:8080/actuator/health`. The `-f` or `--fail` sets the exit return status. These checks are for docker compose only. We can add to OpenShift later.